folly/container/detail/F14Table.h

```diff
@@ -2586,6 +2586,17 @@ class F14Table : public Policy {
 
 #endif // FOLLY_F14_VECTOR_INTRINSICS_AVAILABLE
 
+namespace {
+    const auto erase_if_impl = [](auto& c, auto predicate) {
```

Comment:
This seems like this is not properly namespaced within `f14::detail`.

Reply:
Just moved `erase_if_impl` into `f14::detail` namespace to match local consistency.